rmc/utils/simultaneous_breaks.py

```diff
@@ -410,6 +418,7 @@ def process_section_group(
 
     :param str ht_path: Path to input Table (Table written using `group_no_single_break_found_ht`).
     :param List[str] section_group: List of transcripts or transcript sections to process.
+    :param count: Which transcript group is being run (based on counter generated in `main`).
```

Comment:
```suggestion
    :param count: Which transcript or transcript section group is being run (based on counter generated in `main`).
```
And the same for all other instances of this